compiler/rustc_hir_analysis/src/coherence/orphan.rs

```diff
@@ -296,15 +302,6 @@ fn emit_orphan_check_error<'tcx>(
                 };
 
                 ty = tcx.erase_regions(ty);
-                ty = match ty.kind() {
```

Comment:
Ah lol, thanks :) I encountered this in #117164 and was a bit weirded out by it, glad it's gone.

Reply:
💀